modules/income_and_assets/lib/income_and_assets/pdf_fill/sections/section_06.rb

```diff
@@ -0,0 +1,154 @@
+            key: "F[0].IncomeRecipients6[#{ITERATOR}]"
+          },
+          'recipientRelationshipOverflow' => {
+            question_num: 6,
+            question_suffix: '(1)',
+            question_text: "SPECIFY INCOME RECIPIENT'S RELATIONSHIP TO VETERAN"
+          },
+          'otherRecipientRelationshipType' => {
+            key: "F[0].OtherRelationship6[#{ITERATOR}]",
+            question_num: 6,
+            question_suffix: '(1)',
+            question_text: 'RELATIONSHIP TYPE OTHER'
+          },
+          # Q2
+          'recipientName' => {
+            key: "F[0].NameofIncomeRecipient6[#{ITERATOR}]",
+            limit: 37,
+            question_num: 6,
+            question_suffix: '(2)',
+            question_text:
+              'SPECIFY NAME OF INCOME RECIPIENT (Only needed if Custodian of child, child, parent, or other)'
+          },
+          # Q3
+          'incomeGenerationMethod' => {
+            key: "F[0].HowIncomeIsGenerated6[#{ITERATOR}]"
+          },
+          'incomeGenerationMethodOverflow' => {
+            question_num: 6,
+            question_suffix: '(3)',
+            question_text: 'SPECIFY HOW INCOME IS GENERATED'
+          },
+          'otherIncomeType' => {
+            limit: 73,
+            question_num: 6,
+            question_suffix: '(3)',
+            question_text: 'INCOME TYPE OTHER',
+            key: "F[0].OtherIncomeGenerationMethod6[#{ITERATOR}]"
+          },
+          # Q4
+          'grossMonthlyIncome' => {
+            'thousands' => {
+              key: "F[0].GrossMonthlyIncome1_6[#{ITERATOR}]"
+            },
+            'dollars' => {
+              key: "F[0].GrossMonthlyIncome2_6[#{ITERATOR}]"
+            },
+            'cents' => {
+              key: "F[0].GrossMonthlyIncome3_6[#{ITERATOR}]"
+            }
+          },
+          'grossMonthlyIncomeOverflow' => {
+            question_num: 6,
+            question_suffix: '(4)',
+            question_text: 'GROSS MONTHLY INCOME'
+          },
+          # Q5
+          'fairMarketValue' => {
+            'millions' => {
+              key: "F[0].FairMarketValue1_6[#{ITERATOR}]"
+            },
+            'thousands' => {
+              key: "F[0].FairMarketValue2_6[#{ITERATOR}]"
+            },
+            'dollars' => {
+              key: "F[0].FairMarketValue3_6[#{ITERATOR}]"
+            },
+            'cents' => {
+              key: "F[0].FairMarketValue4_6[#{ITERATOR}]"
+            }
+          },
+          'fairMarketValueOverflow' => {
+            question_num: 6,
+            question_suffix: '(5)',
+            question_text: 'SPECIFY FAIR MARKET VALUE OF THIS ASSET'
+          },
+          # Q6
+          'canBeSold' => {
+            key: "F[0].CanAssetBeSold6[#{ITERATOR}]"
+          },
+          'canBeSoldOverflow' => {
+            question_num: 6,
+            question_suffix: '(6)',
+            question_text: 'CAN THIS ASSET BE SOLD?'
+          }
+        }
```

Comment:
Will add Q7 back in the next PR, just needed something to get under the LOC limit